infra/Makefile

```diff
@@ -0,0 +1,7 @@
+ENVIRONMENT?=staging
+
+init:
+	cd $(ENVIRONMENT) && terraform init
+
```

Comment:
Suggestion: Add `&& cp secret.tfvars.template secret.tfvars`, which already gives us a file we can mess around with :)

Reply:
https://github.com/bettersg/call-home/pull/87/commits/d082e99765c55115663117c99cc9703b30ad73e2